filebeat/prospector/prospector_log.go

```diff
@@ -18,14 +18,16 @@ var (
 	filesTruncated = monitoring.NewInt(nil, "filebeat.prospector.log.files.truncated")
 )
 
-type ProspectorLog struct {
+// Log contains the prospector and its config
+type Log struct {
```

Comment:
"type name will be used as prospector.ProspectorLog by other packages, and that stutters; consider calling this Log (golint)"